test/redis_client_mocks/nearby_search_test.go

```diff
@@ -71,23 +71,51 @@ func init() {
 	}
 }
 
-// The setup of this test case guarantees that the Peter Luger's Steakhouse is located OUTSIDE the search radius coverage
+// The setup of this test case guarantees that the Peter Luger's Steakhouse is located OUTSIDE the search radius coverage,
 func TestGetPlaces_shouldExcludePlacesOutsideOfSearchRadius(t *testing.T) {
 	placeSearchRequest := iowrappers.PlaceSearchRequest{
-		Location: POI.Location{Longitude: -74.0060, Latitude: 40.7128},
-		PlaceCat: POI.PlaceCategoryEatery,
-		Radius:   uint(5000),
+		Location:   POI.Location{Longitude: -74.0060, Latitude: 40.7128},
+		PlaceCat:   POI.PlaceCategoryEatery,
+		Radius:     uint(5000),
+		PriceLevel: POI.PriceLevelFour,
 	}
 
 	cachedEateryPlaces, err := RedisClient.NearbySearch(RedisContext, &placeSearchRequest)
 	if err != nil {
 		t.Errorf(err.Error())
 		return
 	}
+	// "Keens Steakhouse"
+	var expect_place = places[2]
```

Comment:
Nit: use snakeCase for variable names. Also consider `expectedPlace`.

Reply:
Done, changed in [ab993eb](https://github.com/timwangmusic/Vacation-Planner/pull/282/commits/ab993ebd4311a64a4d8dbb785bed8439be526203)